src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
> They're not "settings". The user has no explicit interaction with these state variables.... Having the window position reset to the default position once in 5 years (if ever) is hardly going to ruin a DJs career lol. 

We use the mixxx.cfg file to store user state -- for example the state of different skin features (whether we're in 4 deck mode versus 2 deck mode), and the last position of the library sidebar separator. Whether you call this a "setting" or a "preference" is semantics -- it's all user state that we're persisting.

> I still don't see where you're imagining this maintenance burden to be...

Mixxx has a few standard ways of storing structured user state. One is the library (a SQLite database), one is the sound device configuration file, and one is the Mixxx config file. You're proposing we add another configuration file -- one that may not even be located within the user's profile folder due to Qt's choice of where to store QSettings files.

For me, that already plainly spells out unnecessary maintenance burden. I've been through a lot of hairy profile migrations while releasing the past 7 versions of Mixxx. I'd like to keep the user profile as simple as we can.